data/json/hobbies.json

```
@@ -91,5 +91,208 @@
+    "skills": [ { "level": 1, "name": "computer" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "trapping",
+    "name": "Trapping",
+    "description": "You like to trap little wild animals such as mice and other rodents.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "traps" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "dodgeball",
+    "name": "Dodgeball",
+    "description": "Dodging fast moving balls as if your life depended on it was fun.  Zombies aren't as fun, but your life certainly depends on it.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "dodge" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "amateur_electronics",
+    "name": "Amateur Electronics",
+    "description": "You found unexpected solace while soldering components to circuit boards.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "electronics" }, { "level": 2, "name": "computer" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "diy_crafts",
+    "name": "DIY Crafts",
+    "description": "You spent much of your time making something useful out of bits and bobbles.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "fabrication" }, { "level": 1, "name": "tailor" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "cooking",
+    "name": "Cooking",
+    "description": "You enjoyed cooking for yourself and others.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "cooking" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "paintball",
+    "name": "Paintball",
+    "description": "Shooting your enemies has always been fun.  Now you can do it for real.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "rifle" }, { "level": 1, "name": "gun" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "plinking",
+    "name": "Plinking",
+    "description": "You enjoyed plinking with your airsoft pistol.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "pistol" }, { "level": 1, "name": "gun" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "skeet_shooting",
+    "name": "Skeet Shooting",
+    "description": "Where the skeet go, your bullet follows.  Or atleast, one of them will.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "shotgun" }, { "level": 1, "name": "gun" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "socializing",
+    "name": "Public Speaking",
+    "description": "You were an incredibly social individual.",
+    "points": 1,
+    "skills": [ { "level": 3, "name": "speech" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "street_fighting",
+    "name": "Street Fighting",
+    "description": "You were always instigating fights in the street.  Atleast this might come in useful now.",
+    "points": 1,
```

```suggestion
    "points": 2,
```
Should probably also grant the brawling martial art?

---

Isn't brawling granted by default?

---

It is granted by default, yes. 😛